content/sensu-go/5.14/api/events.md

```diff
@@ -690,8 +651,7 @@ curl -X PUT \
   "entity": {
     "entity_class": "proxy",
     "metadata": {
-      "name": "server1",
-      "namespace": "default"
```

Comment:
[Per the sensu-go changelog](https://github.com/sensu/sensu-go/blob/master/CHANGELOG.md#5170---2020-01-28), I believe the removal of namespace as a minimum required attribute should only apply to 5.17 and later.